resources/js/components/Card.vue

```diff
@@ -1,70 +1,81 @@
 <template>
-    <card class="overflow-hidden flex flex-row">
-        <button
-                v-for="option in filter.options"
-                :key="option.value"
-                class="py-4 px-8 border-b-2 focus:outline-none flex-1"
-                :class="[isActive(option) ? 'text-grey-black font-bold border-primary': 'text-grey font-semibold border-40']"
-                @click="handleChange(option)"
-        >{{ option.name }}
-        </button>
-    </card>
+  <card class="overflow-hidden flex flex-row">
+    <button
+      v-for="option in filter.options"
+      :key="option.value"
+      class="py-4 px-8 focus:outline-none flex-1"
+      :class="[
+        isActive(option)
+          ? 'border-b-2 border-primary-500 text-primary-500 bold'
```

Comment:
```suggestion
          ? 'border-b-2 border-primary-500 text-primary-500 font-bold'
```
"bold" didn't work for me, but "font-bold" does.